routing.rst

```diff
@@ -136,6 +136,89 @@ use them later to :ref:`generate URLs <routing-generate>`.
     configure your routes in YAML, XML or PHP, that's no problem! Just create a
     new routing file (e.g. ``routes.xml``) and Symfony will automatically use it.
 
+.. _i18n-routing:
+
+Localized Routing (i18n)
+------------------------
+
+Routes can be localized to provide unique paths per *locale*. Symfony provides a
+handy way to declare localized routes without duplication.
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        // src/Controller/BlogController.php
+        namespace App\Controller;
+
+        use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+        use Symfony\Component\Routing\Annotation\Route;
+
+        class BlogController extends Controller
+        {
+            /**
+             * Matches /blog exactly
+             *
+             * @Route("/blog", name="blog_list")
```

Comment:
The annotation example look wrong to me. It should have the same routes as the YAML, XML, and PHP examples.